app/src/main/java/xyz/malkki/neostumbler/geosubmit/GeosubmitParams.kt

```diff
@@ -10,8 +10,7 @@ data class GeosubmitParams(
     val apiKey: String?
 ) {
     companion object {
-        //NOTE: MLS does not accept data submissions anymore, this is used as a default just to show an example
-        const val DEFAULT_BASE_URL = "https://location.services.mozilla.com"
+        const val DEFAULT_BASE_URL = "https://beacondb.net"
```

Comment:
@joelkoen requested on Matrix that defaults would use the api subdomain, which while functionally identical, will allow scaling or something if necessary, so I think changing the default here should also use it.

```suggestion
        const val DEFAULT_BASE_URL = "https://api.beacondb.net"
```

Reply:
ah good catch, thanks!